content/en/developer-guide/first_steps.md

```diff
@@ -0,0 +1,51 @@
+### Running Localstack
+
+Once you have cloned the localstack repository, and setup the environment, with dependencies installed,
+You should be able to create the python virtualenv with
+
+  ```bash
```

Comment:
it would be great if we could use the `{{< command >}}` shortcode for the commands, see https://github.com/localstack/docs/pull/55 for details.

Reply:
It would be great if you can make this change at all of the place where you have referenced the commands 😄

Reply:
Amazing, Thanks @HarshCasper , working on it.